spec/features/users_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+feature "Guest interactions" do
+  before do
+    visit root_path
+  end
+
+  scenario "guests are brought to sign up page" do
+    expect(page).to have_content "Connect with all your friends!"
+  end
+
+  scenario "guests cannot sign in without entering a proper login and email" do
+    click_button "Sign In"
+    expect(page).to have_content "Connect with all your friends!"
+  end
+
+  scenario "guests can sign up" do
```

Comment:
Woah..... I'd refactor the filling out of forms in these following tests into a macro.

Consider a method that takes **args or an options hash, and fills in forms with either args[:option] || "my_default_option" for each form field.